packages/my-react-package/src/MetaWallet.tsx

```diff
@@ -25,6 +26,9 @@ class MetaWallet {
     TxBuilders: {
         [key: string]: string
     };
+    authorizeEndpoints: {
```

Comment:
If we'll use "Endpoints" keyword here, let's use it in 'TxBuilders' and 'toGasStation' as well just to stay consistent.